tools/daemon/flow.c

```diff
@@ -508,9 +511,9 @@ int del_flow(struct store_pid *pid_value, struct store_flow *value)
 				rc = -EFAULT;
 			}
 
-			bitmap_destroy(cur_element->ctx->ht);
 			assert(ctx == cur_element->ctx);
 			free_pending_list(pid, cur_element->ctx, value->if_id);
+                        bitmap_destroy(cur_element->ctx->ht);
```

Comment:
Why do we move it ?

Reply:
> Why do we move it ?

`free_pending_list()` can access `ht` https://github.com/Mellanox-lab/libxlio/blob/master/tools/daemon/flow.c#L654